store/instance.go

```diff
@@ -404,19 +404,19 @@ func (s *Store) createInstanceRaw(ctx context.Context, create *InstanceCreate) (
 
 	for _, dataSource := range create.DataSourceList {
 		dataSourceCreate := &api.DataSourceCreate{
-			CreatorID:    create.CreatorID,
-			InstanceID:   instance.ID,
-			DatabaseID:   allDatabase.ID,
-			Name:         dataSource.Name,
-			Type:         dataSource.Type,
-			Username:     dataSource.Username,
-			Password:     dataSource.Password,
-			SslKey:       dataSource.SslKey,
-			SslCert:      dataSource.SslCert,
-			SslCa:        dataSource.SslCa,
-			HostOverride: dataSource.HostOverride,
-			PortOverride: dataSource.PortOverride,
-			Options:      dataSource.Options,
+			CreatorID:  create.CreatorID,
+			InstanceID: instance.ID,
+			DatabaseID: allDatabase.ID,
+			Name:       dataSource.Name,
+			Type:       dataSource.Type,
+			Username:   dataSource.Username,
+			Password:   dataSource.Password,
+			SslKey:     dataSource.SslKey,
+			SslCert:    dataSource.SslCert,
+			SslCa:      dataSource.SslCa,
+			Host:       dataSource.Host,
+			Port:       dataSource.Port,
+			Options:    dataSource.Options,
```

Comment:
So here, we are not assigning the database name field added in the data_source table. Is this intended?

Reply:
Done.

Was trying to use a different PR.